backend/src/server_process_helper.py

```diff
@@ -185,7 +185,7 @@ async def get_sse(self, request: Request):
             "/sse",
             headers=request.headers,
             data=request.body,
-            timeout=None,
+            timeout=5,
```

Comment:
This seems to have fixed the issue, and I don't think it's caused any side effects